### PR TITLE
Documentation/android spinner

### PR DESCRIPTION
**Pull request by Cheesebaron**
### :sparkles: What kind of change does this PR introduce? (Bug fix, feature, docs update...)
docs

### :arrow_heading_down: What is the current behavior?
no docs

### :memo: Links to relevant issues/docs
#2631 

### :thinking: Checklist before submitting

- [ ] All projects build
- [ ] Follows style guide lines ([code style guide](https://github.com/MvvmCross/MvvmCross#code-style-guidelines))
- [x] Relevant documentation was updated ([docs style guide](https://www.mvvmcross.com/documentation/contributing/mvvmcross-docs-style-guide))
- [x] Rebased onto current develop

### CODE REVIEW

File: docs/_documentation/platform/android/android-recyclerview.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Android MvxRecyclerView
 category: Platforms
 ---
 
-Available in Android Support RecyclerView, MvvmCross 5.
+Available in Android Support RecyclerView, MvvmCross 4.
```
Comment (nickrandolph) [inline docs/_documentation/platform/android/android-recyclerview.md]: In android-spinner.md it says "Available in MvvmCross 3." - should the numbering be consistent as here it says 4?

Comment (Cheesebaron) [reply docs/_documentation/platform/android/android-recyclerview.md]: RecyclerView was added later than the Spinner, why should it be consistent? This indicates in which version of MvvmCross this stuff is available.

Comment (nickrandolph) [reply docs/_documentation/platform/android/android-recyclerview.md]: Your explanation make sense with regards to the different version numbers.

Comment (Cheesebaron) [reply docs/_documentation/platform/android/android-recyclerview.md]: 👍

**Pull request merged**